### PR TITLE
Installing marionette

### PR DESCRIPTION
**Pull request by scott-w**
I've written a really basic Marionette install guide to help with #7 

I've just covered the case that I understand - installing with Browserify - but I'd really like to get a few alternatives in place with a decision on the primary install method.

### CODE REVIEW

File: en/getting_started/installing_marionette.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+## Using NPM and Browserify
+
+To install Marionette using NPM, we must install a few dependencies and then
+install Marionette itself.
+
+  1. Install NPM following the advice from the [NPM blog][install-npm]
+  2. Create a directory for your JavaScript application
+  3. Inside that directory, run `npm init`, giving your application names
+  4. Install Browserify: `sudo npm install -g browserify`
+  5. Install Underscore: `npm install --save underscore@1.7.0`
```
Comment (paulfalgout) [inline en/getting_started/installing_marionette.md]: why `1.7.0`?  latest underscore works right?

Comment (scott-w) [inline en/getting_started/installing_marionette.md]: I'll test it, the last time I tried the latest underscore I ran into some difficulties with the setup.

Comment (samccone) [inline en/getting_started/installing_marionette.md]: I think there was a regression in 1.7.x but it was fixed... however I defer to @jridgewell

Comment (scott-w) [inline en/getting_started/installing_marionette.md]: I'll pull this into a separate issue so I don't forget to look at it.

**Comment by ruwhan** [comment]
hello Scott, I have a question. How do you make `template: require('./templates/layout.html')` work at the first code snippet in the book. Thanks.

And thanks for the guide.

**Comment by scott-w** [comment]
@ruwhan You pass `-t node-underscorify` as one of the arguments to `browserify`. The Getting Started guide should have some of the detail there.

**Comment by ruwhan** [comment]
thanks for quick reply @scott-w , :+1: 

`-t node-browserify` is  not working either, I need to add browserify configuration in `package.json` file to make it working, and pass no argument for `browserify`. Hope its helps. :smile: 

```javacsript
{
    "browserify": {
        "transform": [
            ["node-underscorify", {
                "extensions": ["html"],
                "requires": [{"variable": "_", "module": "underscore"}]
            }]
        ]
    }
}
```

**Comment by scott-w** [comment]
Thanks! Feel free to stick that directly into the guide where the `package.json` file is described.

**Pull request merged**